doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -471,6 +471,39 @@ Field              | Type       | Meaning
 `cpu_nano_cores`   | [`bigint`] | Approximate CPU usage, in billionths of a vCPU core.
 `memory_bytes`     | [`bigint`] | Approximate RAM usage, in bytes.
 
+### `mz_storage_source_statistics`
```

Comment:
"storage source" is not the right name! Either "storage host" or "source". But I think having two separate views—`mz_source_statistics` and `mz_sink_statistics`—is best, since we plan to keep the general structure of these views even after transitioning them to introspection views, and then they'll be consistent with e.g. `mz_{source,sink}_status`.